.pre-commit-config.yaml

```diff
@@ -56,39 +52,11 @@ repos:
       - id: blacken-docs
         additional_dependencies: [black==22.8.0]
 
-  - repo: https://github.com/PyCQA/isort
-    rev: 5.11.4
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.226
     hooks:
-      - id: isort
-
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.3.1
-    hooks:
-      - id: pyupgrade
-        args: ["--py37-plus"]
-
-  - repo: https://github.com/hadialqattan/pycln
-    rev: v2.1.3
-    hooks:
-      - id: pycln
-        args: [--all]
-        stages: [manual]
-
-  - repo: https://github.com/asottile/yesqa
-    rev: v1.4.0
-    hooks:
-      - id: yesqa
-        exclude: docs/conf.py
-        additional_dependencies: &flake8_dependencies
-          - flake8-bugbear
-          - flake8-print
-
-  - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
-    hooks:
-      - id: flake8
-        exclude: docs/conf.py
-        additional_dependencies: *flake8_dependencies
+      - id: ruff
+        args: ["--fix"]
```

Comment:
This really should have --force-exclude, but the next version of the Action won't need it. https://github.com/charliermarsh/ruff-pre-commit/pull/20

Reply:
And it's out!